cmd/notation/verify.go

```diff
@@ -112,7 +114,11 @@ func runVerify(command *cobra.Command, opts *verifyOpts) error {
 			fmt.Printf("Warning: %v was set to \"logged\" and failed with error: %v\n", result.Type, result.Error)
```

Comment:
Out of scope for this PR, but we should be printing the outcome details of ALL signatures, not just the first one.

Reply:
We could have this discussion post RC.1, but in short, we decided to only return verify outcome for the succeeded verified signature and early break. On failure, we will use the -v and -d flags to show the details of each signature.